src/lib/Menu/UserMenuBuilder.php

```diff
@@ -89,9 +89,14 @@ public function createStructure(array $options): ItemInterface
             );
 
             $menu->addChild(
-                $this->createMenuItem(self::ITEM_LOGOUT, ['route' => 'logout', 'extras' => [
+                $this->createMenuItem(self::ITEM_LOGOUT, [
+                    'route' => 'logout',
+                    'attributes' => [
+                        'class' => 'ibexa-popup-menu__item--with-border',
+                    ],
+                    'extras' => [
                     'orderNumber' => 60,
-                ]])
+                    ], ])
```

Comment:
I think it's wrong formatting? It should be in two lines

Reply:
I used the same formatting as above.

Reply:
Shouldn't `'orderNumber' => 60,` be indented?